src/oge/gross_to_net_generation.py

```diff
@@ -64,16 +67,16 @@ def convert_gross_to_net_generation(cems, eia923_allocated, plant_attributes, ye
         cems["gross_generation_mwh"] * cems["annual_plant_ratio"]
     )
 
-    cems.loc[
-        cems["net_generation_mwh"].isna(), "gtn_method"
-    ] = "3_annual_subplant_shift_factor"
+    cems.loc[cems["net_generation_mwh"].isna(), "gtn_method"] = (
+        "3_annual_subplant_shift_factor"
```

Comment:
why is this getting reformatted to ("string")? Is this the standard ruff formatting?

In general it seems like there were a lot of formatting updates, which is surprising to me since the whole repo had already been formatted with ruff...

Reply:
It comes from the new 0.3.0 ruff